### PR TITLE
FIX: Drag thresholds in UI deactivated by Sliders (case 1275834).

### PR DESCRIPTION
**Pull request by Rene-Damm**
Fixes [1275834](https://issuetracker.unity3d.com/issues/inputsystem-drag-threshold-value-is-ignored-for-scroll-view-after-interacting-with-a-slider-slash-scroll-bar) ([internal](https://fogbugz.unity3d.com/f/cases/1275834/)).

### CODE REVIEW

File: Packages/com.unity.inputsystem/InputSystem/Editor/AssetEditor/InputActionTreeViewItems.cs [hunk 1]
```diff
@@ -239,8 +239,21 @@ public ActionTreeItem(SerializedProperty actionMapProperty, SerializedProperty a
...
+                if (!string.IsNullOrEmpty(expectedControlType))
+                    return expectedControlType;
+
+                var type = property.FindPropertyRelative("m_Type").intValue;
+                if (type == (int)InputActionType.Button)
+                    return "Button";
+
+                return null;
+            }
+        }
```
Comment (Rene-Damm) [inline Packages/com.unity.inputsystem/InputSystem/Editor/AssetEditor/InputActionTreeViewItems.cs:256]: Unrelated change. Tweak to a previous PR. Just couldn't be bothered to make this a separate PR...

**Review by karljj1 — APPROVED** [review]
